desc/geometry/core.py

```diff
@@ -45,6 +45,7 @@ def compute(
         params=None,
         transforms=None,
         data=None,
+        override=True,
```

Comment:
can we make this `override_grid`? just a generic `override` is not the most informational keyword